### DISCUSSION TITLE
How to know the length of fields

### DISCUSSION THREAD

**Discussion by danimpm** [trakt/trakt-api#276] [Q&A] [answered]
Hi!

This is the first time that I'm using an API with Apiary documentation, and I can't find how to know the response objects lengths. 
I can extract the data type from the examples, but I need to know the maximum length of each field in order to adjust my database with that values.

For example, on the Shows Summary:

https://api.trakt.tv/shows/id

```javascript
{
  "title": "Game of Thrones",
  "year": 2011,
  "ids": {
    "trakt": 353,
    "slug": "game-of-thrones",
    "tvdb": 121361,
    "imdb": "tt0944947",
    "tmdb": 1399
  }
}
```
How can I know the maximum length of the slug id?

Sorry if there is a simple way to get this information, but I can't find it...

Thank you in advance!

Daniel.

**Comment by rectifyer** [comment, marked as answer]
In general it’s going to be 255 characters, the max for a VARCHAR in your database. Some fields are longer TEXT fields like overviews and descriptions.
